app/config/routing.yml

```diff
@@ -9,9 +9,9 @@ app:
     type:     annotation
     prefix:   /{_locale}
     requirements:
-        _locale: %app_locales%
+        _locale: '%app_locales%'
```

Comment:
``%`` is not special in Yaml 2, so quoting is not necessary. The symfony-docs quotes it all the time because the Pygments highlighter follows the rule of Yaml 1.1 where quoting was needed